### PR TITLE
Implement CaaS plugin

### PR DESCRIPTION
**Pull request by gboutry**
Add Container Infra plugin, and documentation.

### CODE REVIEW

File: sunbeam-python/sunbeam/plugins/container_infra/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Installation
+
+To enable the Container Infra service, you need an already bootstraped Sunbeam instance and the following plugins: Secrets, Orchestration, Loadbalancer (optional). Then, you can install the plugin with:
+
+```bash
+sunbeam enable container-infra
+```
+
+## Contents
```
Comment (gboutry) [inline sunbeam-python/sunbeam/plugins/container_infra/README.md:13]: Missing configure section

Comment (gboutry) [reply sunbeam-python/sunbeam/plugins/container_infra/README.md:13]: Also, add plugin configure to main configure

Comment (gboutry) [reply sunbeam-python/sunbeam/plugins/container_infra/README.md:13]: Move configure to `sunbeam configure container-infra`

**Comment by gboutry** [comment]
Force push is rebase on main, moving magnum to 2023.2/edge

**Comment by gboutry** [comment]
Draft, rename to caas ?

**Review by hemanthnakkina — APPROVED** [review]

**Pull request merged**